tsdb/head.go

```diff
@@ -1894,8 +1912,11 @@ type memSeries struct {
 	//
 	// pN is the pointer to the mmappedChunk referered to by HeadChunkID=N
 	mmappedChunks []*mmappedChunk
-	headChunk     *memChunk          // Most recent chunk in memory that's still being built.
-	firstChunkID  chunks.HeadChunkID // HeadChunkID for mmappedChunks[0]
+	// Most recent chunks in memory that are still being built or waiting to be mmapped.
+	// This is a linked list, headChunks points to the most recent chunk, headChunks.next points
+	// to older chunk and so on.
+	headChunks   *memChunk
```

Comment:
Decided to rename it from `headChunk` to `headChunks` because:
A) it's now a list so there can be multiple chunks
B) it flags on this PR all places where `headChunk` was used, which I hope might help with review

Reply:
Good idea!